docs/uml/flintmodel-model.puml

```diff
@@ -0,0 +1,55 @@
+@startuml FlintModel-Model
+skinparam nodesep 50
+skinparam ranksep 150
+
+interface "Flint.structure.IFlintModel" <INamedConcept>
```

Comment:
include these methods in IFlintModel

![image](https://user-images.githubusercontent.com/15971736/182332466-5179027a-9937-48b5-9c63-fb8e36c7502b.png)